keyboards/preonic/keymaps/trigotometry/keymap.c

```diff
@@ -0,0 +1,204 @@
+#include "preonic.h"
+#include "action_layer.h"
+#ifdef AUDIO_ENABLE
+  #include "audio.h"
+#endif
+
+// Layer declarations
+enum preonic_layers {
+  _DVORAK = 0,
+  _QWERTY = 1,
+  _GAMING = 2,
+  _LOWER = 3,
+  _RAISE = 4,
+};
+
+enum preonic_keycodes {
+  DVORAK = SAFE_RANGE,
+  QWERTY,
+  GAMING,
+  LOWER,
+  RAISE,
+};
+
+// QMK predefined macros
+#define SFT_ENT   SFT_T(KC_ENT)   // Tap for Enter, hold for Shift
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31